recipes/DocExtract.py

```diff
@@ -94,7 +100,10 @@ class RequestModelBase(BasePage.RequestModel):
         sheet_url: FieldHttpUrl | None
 
         selected_asr_model: typing.Literal[tuple(e.name for e in AsrModels)] | None
-        # language: str | None
+        language: str | None
+        translation_model: (
+            typing.Literal[tuple(e.name for e in TranslationModels)] | None
+        )
         google_translate_target: str | None
```

Comment:
`translation_target` not added to request model, deprecate `google_translate_target` as done in  `AsrPage` by subclassing `TranslationOptions`